-1

**Failed Tests**:   Build 
**Summary**: https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-77b024/15858/summary.html
**COMMIT**: 7387218eff19f71af99ec72ae5eefcfda5d1f8a9
**CMSSW**: CMSSW_12_0_X_2021-06-09-2300/slc7_amd64_gcc10
**User test area**: For local testing, you can use `/cvmfs/cms-ci.cern.ch/week0/cms-sw/cmssw/34076/15858/install.sh` to create a dev area with all the needed externals and cmssw changes.


## Build

I found compilation error when building: 

<pre>>> Compiling edm plugin /data/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_12_0_X_2021-06-09-2300/src/CondTools/SiStrip/plugins/SiStripSummaryReader.cc
>> Compiling edm plugin /data/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_12_0_X_2021-06-09-2300/src/CondTools/SiStrip/plugins/SiStripThresholdBuilder.cc
>> Compiling edm plugin /data/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_12_0_X_2021-06-09-2300/src/CondTools/SiStrip/plugins/SiStripThresholdReader.cc
>> Building  edm plugin tmp/slc7_amd64_gcc10/src/CondTools/SiStrip/plugins/CondToolsSiStripPlugins/libCondToolsSiStripPlugins.so
/cvmfs/cms-ib.cern.ch/nweek-02684/slc7_amd64_gcc10/external/gcc/10.2.0/bin/../lib/gcc/x86_64-unknown-linux-gnu/10.2.1/../../../../x86_64-unknown-linux-gnu/bin/ld: cannot find -lCommonToolsTrackerMap
collect2: error: ld returned 1 exit status
gmake: *** [tmp/slc7_amd64_gcc10/src/CondTools/SiStrip/plugins/CondToolsSiStripPlugins/libCondToolsSiStripPlugins.so] Error 1
Leaving library rule at src/CondTools/SiStrip/plugins
------- copying files from src/CondTools/SiStrip/scripts -------
>> copied CreatingTables.sh
>> copied SiStripDAQPopCon.py
</pre>

--- a/CalibFormats/SiStripObjects/interface/SiStripDetInfo.h
+++ b/CalibFormats/SiStripObjects/interface/SiStripDetInfo.h
@@ -23,7 +23,7 @@
 // user include files
 #include <map>
 #include <vector>
-#include <stdint.h>
+#include <cstdint>
 // forward declarations
 
 class SiStripDetInfo {

